resources/application.yaml

```diff
@@ -1,94 +1,29 @@
+# Documentation about this file can be found in this file : /docs/install/1-CONFIG.md
```

Comment:
Remove space before ":", surround the path with backticks and drop the first "/".